dmapiclient/data.py

```diff
@@ -20,30 +20,26 @@ def find_audit_events(
             acknowledged=None,
             object_type=None,
             object_id=None,
+            user=None,
```

Comment:
This change would be more backward-compatible if the new argument was added last in the list?

Reply:
That it would, sir.

(though I would be more than a little worried if people were using >= 8 keyword arguments positionally)

Reply:
Technically API breaking change and should be a major version bump. :trollface: